src/scss/dp/overrides/components/_timeseries-explorer.scss

```diff
@@ -31,26 +37,19 @@
 .timeseries__list {
   max-height: 464px;
   overflow-x: hidden;
-  margin-left: 16px;
 }
 
 .list--neutral {
   list-style: none;
-  padding: 0;
+  padding-left: 0
+
 }
 
 .timeseries__remember {
   display: none;
-  margin-bottom: 16px;
-  padding: 6px 16px 2px;
+  border-bottom: 1px solid #d0d2d3;
 }
 
 .timeseries__download {
-  border-top: 1px solid #d0d2d3;
-  margin: -1px 16px 0;
   display: none;
 }
-
-.border-bottom {
-  border-bottom: 1px solid #d0d2d3;
-}
```

Comment:
EOF

Reply:
Are you saying I *shouldn't* have a blank line at the end of this file?